SPHINXsys/src/for_3D_build/high_level_simulation_class/solid_structural_simulation_class.cpp

```diff
@@ -510,3 +510,44 @@ void StructuralSimulation::RunSimulation(Real end_time)
 	tt = t4 - t1 - interval;
 	cout << "Total wall time for computation: " << tt.seconds() << " seconds." << endl;
 }
+
+void StructuralSimulation::InitSimulationJS()
```

Comment:
Can you just make it to InitSimulation? And remove it from RunSimulation, the same way you removed it here and move it into the very end of the StructuralSimulation class constructor? So we don't need InitSimulationJS? I guess everything initialization should be in the constructor, looks nicer too